### ISSUE TITLE
Request for MAI2021 Validation/Test Dataset

### ISSUE THREAD

**Issue by kaiq663** [rflepp/SplitterNet-Efficient-Mobile-Denoising-Models-CVPR2024#3]
After preprocessing the MAI2021 training dataset by cropping the  images to 1024x1024 resolution, I am left with 7868 images, which I believe is too large for a validation or test set.

Since MAI2021 does not provide validation and test set in their official website, would it be possible for you to provide the  validation or test set of MAI2021 used in your research?

**Comment by rflepp** [comment]
You can find it on https://competitions.codalab.org/competitions/28120 after registering for the competition.
